test/account_api_test.rb

```diff
@@ -0,0 +1,278 @@
+require "test_helper"
+require "gds_api/account_api"
+
+describe GdsApi::AccountApi do
+  include PactTest
+
+  let(:api_client) { GdsApi::AccountApi.new(account_api_host) }
+
+  let(:authenticated_headers) { { "GOVUK-Account-Session" => govuk_account_session } }
+  let(:govuk_account_session) { "logged-in-user-session" }
```

Comment:
huh, I didn't know we could use let in minitest. I'd assumed that was an Rspec thing!
cool.